kola/tests/ignition/oem.go

```diff
@@ -67,6 +67,9 @@ func init() {
 		Run:         wipeOEM,
 		MinVersion:  semver.Version{Major: 2983},
 		ClusterSize: 1,
+		// `wiping` the OEM file system does not allow the instance to boot.
+		// More details: https://github.com/flatcar-linux/Flatcar/issues/514.
+		ExcludePlatforms: []string{"esx", "azure"},
```

Comment:
Maybe limiting it to the qemu platform is the best because it doesn't require a working OEM partition. It also helps keeping the testing time down on the cloud platforms.